app/init/websocket_manager.ts

```diff
@@ -186,20 +187,23 @@ class WebsocketManager {
         }
 
         this.cancelAllConnections();
-        if (appState === 'background' && !this.isBackgroundTimerRunning) {
+        if (appState !== 'active' && !this.isBackgroundTimerRunning) {
             this.isBackgroundTimerRunning = true;
             this.cancelAllConnections();
-            BackgroundTimer.runBackgroundTimer(() => {
+            this.backgroundIntervalId = BackgroundTimer.setInterval(() => {
```

Comment:
If we are stopping the in the first loop, shouldn't we use `setTimeout`? Does `setTimeout` exist in BackgroundTimer?